packages/blade/src/components/BottomNav/types.ts

```diff
@@ -0,0 +1,65 @@
+import type { StyledPropsBlade } from '~components/Box/styledProps';
+import type { IconComponent } from '~components/Icons';
+import type { TestID } from '~utils/types';
+
+type BottomNavProps = {
+  /**
+   * children slot of BottomNav, accepts BottomNavItem
+   */
+  children: React.ReactElement[];
+
+  /**
+   * zIndex of BottomNav
+   *
+   * @default 100
+   */
+  zIndex?: number;
+} & StyledPropsBlade &
+  TestID;
+
+type BottomNavItemProps = {
+  /**
+   * Title text of the BottomNavItem
+   */
+  title: string;
+
+  /**
+   * Icon rendered above the title.
+   *
+   * Accepts icon component from blade
+   */
+  icon: IconComponent;
+
+  /**
+   * href property of link
+   *
+   * maps to `to` property when react router is being used
+   */
+  href?: string;
```

Comment:
add link related props like `target`

Reply:
I did think of it during API decision. I feel we shouldn't because BottomNav shouldn't open links in new window

Reply:
makes sense

Reply:
done